Readme.md

```diff
@@ -86,33 +116,41 @@ paths:
 ```
-The reference prices are updated every 10 minutes until the beginning of the delivery hour.
+
+The reference prices for each delivery hour are updated every 10 minutes until gate closure. The updates are sometimes published with some delay, usually something between 0 and 30 seconds but without any clear patterns.
+
+Once the price is available for one delivery hour, it's available for the rest of the tradable hours (That is until the end of the same day).
+
+There is a limit on the number of requests we can submit to the API, about 50 per minute and 1000 per hour.
 
 ## Reporting:
 
 The reporting frontend displays the following dashboard:
 ![img.png](dashboard.png)
-It can be accessed by the traders from their browsers.
-The dashboard is updated with all the current values every 5 Minutes. 
+
+It can be accessed by the traders from their browsers. Since this is the traders' main source of validation, they might get impatient around the time the reference price is supposed to be published and start massively refreshing their reporting dashboard.
+
 
 ## What is expected:
-This is an open challenge, it describes the environment in which this system evolves, the different interfaces with the 
-rest of the world and the functional outcome that we want to achieve. In architectural terms, this would be a black box view:
+This is an open problem, it describes the environment in which this system evolves, the different interfaces with the 
+rest of the world and the functional outcome that we need to achieve. In architectural terms, this would be a black box view:
 ![img_1.png](black_box.png)
 
-In other words, we don't have one solution in mind and there is no right or wrong answer, we are looking forward to a 
-discussion about a solution you think is appropriate.
+In other words, we don't have any particular solution in mind and there is no right or wrong answer, we are looking forward to hear how you would design this system and have a technical discussion about it.
+ 
+Most of this is inspired by actual features and problems we had to or still have to solve, hopefully with your help.
 
-We are interested in how you would design such a system:
+Some aspects that we are particularly interested in are:
 - which components would you need? How do you decide on the boundaries between these components?
-- which databases or any other data management systems?
+- which databases or any other data management systems would you need?
 - how would these components communicate with each other?
 - which technologies would you use for each component and why?
-- How and where would you run such a system? Which cloud services would you use?
-- How would you management updates and deployments in such a system?
-- Anything else that you think is relevant
+- How and where would you run such a system in the cloud? Which services would you use?
+- How would you manageme updates and deployments in such a system?
+- How would you structure logging and monitoring?
```

Comment:
Maybe to much? The other points are much more Important. We want candidates to focus on those, dont we?

Reply:
These are all more like inspirations than actual questions, there is no time to tackle all of these anyway. I think each candidate would focus on some specific aspects, maybe what they know best?